Hazel/src/Platform/Windows/WindowsWindow.h

```diff
@@ -29,7 +29,7 @@ namespace Hazel {
 		virtual void Shutdown();
 	private:
 		GLFWwindow* m_Window;
-		GraphicsContext* m_Context;
+		Hazel::Scope<GraphicsContext> m_Context;
```

Comment:
Could use an std::optional here instead as GraphicsContext doesn't need to be accessed outside this class its just delay initialized (in an ideal situation you wouldn't have two-phase init)